lib/core/test/unit/Cardano/Wallet/Primitive/TypesSpec.hs

```diff
@@ -362,6 +362,14 @@ spec = do
                             . slotStartTime sps
                     Just slot === f slot
 
+        it "slotStartTime (slotRange timeRange) ⊆ timeRange " $
```

Comment:
Small point, but perhaps this should be `"fmap slotStartTime (slotRange timeRange) ⊆ timeRange"`

 What do you think?